charts/kube-otel-stack/templates/collector.yaml

```diff
@@ -44,16 +44,36 @@ spec:
 {{- end }}
   env:
     {{- toYaml $collector.env | nindent 4}}
-    - name: OTEL_RESOURCE_ATTRIBUTES
-      value: "k8s.cluster.name={{ $collector.clusterName | default "unknown" }}"
-    - name: KUBE_NODE_NAME
+    - name: OTEL_KUBE_NODE_NAME
       valueFrom:
         fieldRef:
           fieldPath: spec.nodeName
-    - name: KUBE_POD_NAME
+    - name: OTEL_K8S_NAMESPACE
+      valueFrom:
+        fieldRef:
+          apiVersion: v1
+          fieldPath: metadata.namespace
+    - name: OTEL_K8S_POD_NAME
       valueFrom:
         fieldRef:
+          apiVersion: v1
           fieldPath: metadata.name
+    - name: OTEL_K8S_POD_UID
+      valueFrom:
+        fieldRef:
+          apiVersion: v1
+          fieldPath: metadata.uid
+    - name: OTEL_K8S_POD_IP
+      valueFrom:
+        fieldRef:
+          apiVersion: v1
+          fieldPath: status.podIP
+    - name: OTEL_KUBE_POD_NAME
+      valueFrom:
+        fieldRef:
+          fieldPath: metadata.name
+    - name: OTEL_RESOURCE_ATTRIBUTES
+      value: "k8s.cluster.name={{ $collector.clusterName | default "unknown" }},k8s.pod.ip=$(OTEL_K8S_POD_IP),k8s.pod.uid=$(OTEL_K8S_POD_UID),k8s.namespace.name=$(OTEL_K8S_NAMESPACE),k8s.node.name=$(OTEL_K8S_NODE_NAME),k8s.pod.name=$(OTEL_K8S_POD_NAME)"
```

Comment:
I think OTEL_K8S_NODE_NAME is unset here. i.e. `k8s.node.name=$(OTEL_K8S_NODE_NAME)` should be `k8s.node.name=$(OTEL_KUBE_NODE_NAME)` or `OTEL_KUBE_NODE_NAME` should be renamed